stream/distributedlog/pom.xml

```diff
@@ -47,8 +47,9 @@
         <artifactId>maven-javadoc-plugin</artifactId>
         <version>${maven-javadoc-plugin.version}</version>
         <configuration>
+          <sourcepath>${src.dir}</sourcepath>
+          <notimestamp>true</notimestamp>
           <!-- Avoid for missing javadoc comments to be marked as errors -->
```

Comment:
no longer useful comment

Reply:
@nicoloboschi why? it's commented for `<doclint>none<doclint>`